Oracle Tuxedo Dockerfiles for new version 22.1.0.0.0 (#2478)
oracle/docker-images: OracleTuxedo/core/samples/shm/README.md

> # Tuxedo SHM sample with Container Image

Reviewer: ```suggestion
# Tuxedo SHM sample container image
```

> Example of Image with Tuxedo Domain
> 
> This Dockerfile extends the Oracle Tuxedo image by creating a sample domain.

Reviewer: ```suggestion
This example extends the Oracle Tuxedo image by creating a sample domain.
```

## How to run

> First make sure you have built oracle/tuxedo:latest from the "core" folder.  

Reviewer: ```suggestion
First make sure you have built `oracle/tuxedo:latest` from the "core" folder.  
```

> Now to build this sample, run:  
> docker build -t oracle/tuxedoshm .  
> or  
> ./buildContainerImage.sh

Reviewer: ````suggestion
Now to build this sample, run:

```shell
docker build -t oracle/tuxedoshm .  
```

or  

```shell
./buildContainerImage.sh
```
````

> You can then start the image and run the sample in a new container using the below command :  
> docker run -d -h tuxhost -v ${Local_volumes_dir}:/u01/oracle/user_projects oracle/tuxedoshm

Reviewer: ````suggestion
You can then start the image and run the sample in a new container using the following command:

```shell
docker run -d -h tuxhost -v ${Local_volumes_dir}:/u01/oracle/user_projects oracle/tuxedoshm
```
````
